### PR TITLE
Various Improvements and Fixes

### PR DESCRIPTION
**Pull request by nexpoehler**

### CODE REVIEW

File: playbooks/group_vars/lb-external.yml [hunk 1]
```diff
@@ -58,21 +58,21 @@ ssl_key: |
 ## HAProxy Vars
 haproxy_global:
   log:
-    - address: "/dev/log"
-      facility: "local0"
-      level: "info"
-  stats:
-    socket: "/var/lib/haproxy/stats mode 600 level admin"
+    - address: /dev/log
```
Comment (clwells) [inline playbooks/group_vars/lb-external.yml]: ""

File: playbooks/group_vars/lb-external.yml [hunk 2]
```diff
@@ -58,21 +58,21 @@ ssl_key: |
...
+      facility: local0
+    - address: /dev/log
+      facility: local1
+      level: notice
   ssl_default_bind_options: "{{ ssl_options }}"
   ssl_default_bind_ciphers: "{{ ssl_ciphers }}"
+  chroot: /var/lib/haproxy
```
Comment (clwells) [inline playbooks/group_vars/lb-external.yml]: ""

File: playbooks/tasks/build/sync.yml [hunk 3]
```diff
@@ -2,25 +2,25 @@
 
 - name: Create Build Code Base
   file: 
-    path="{{ hostvars[groups['fpm'][0]]['nex_app_build_code_base'] }}"
+    path="{{ hostvars[groups['fs'][0]]['nex_app_build_code_base'] }}"
     state="directory"
     mode="0711"
```
Comment (clwells) [inline playbooks/tasks/build/sync.yml]: no ""

File: playbooks/tasks/lb-all.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+- name: 'Ensure chroot directory exists'
+  file:
+    name: "{{ haproxy_global.chroot }}/dev"
```
Comment (clwells) [inline playbooks/tasks/lb-all.yml]: XX=""

File: playbooks/tasks/user.yml [hunk 5]
```diff
@@ -49,7 +49,7 @@
     owner="{{ nex_app_unix_user }}"
     group="{{ nex_app_unix_group }}"
     state="directory"
-    mode=2751
+    mode=711
```
Comment (clwells) [inline playbooks/tasks/user.yml]: 0711?

File: playbooks/group_vars/lb-external.yml (hunk 1 above)
Comment (nexpoehler) [reply playbooks/group_vars/lb-external.yml]: done

File: playbooks/group_vars/lb-external.yml (hunk 2 above)
Comment (nexpoehler) [reply playbooks/group_vars/lb-external.yml]: done

File: playbooks/tasks/build/sync.yml (hunk 3 above)
Comment (nexpoehler) [reply playbooks/tasks/build/sync.yml]: done

File: playbooks/tasks/lb-all.yml (hunk 4 above)
Comment (nexpoehler) [reply playbooks/tasks/lb-all.yml]: done

File: playbooks/tasks/user.yml (hunk 5 above)
Comment (nexpoehler) [reply playbooks/tasks/user.yml]: done

**Pull request merged**